### DISCUSSION TITLE
automated dev binary

### DISCUSSION THREAD

**Discussion by ZyanKLee** [noisetorch/NoiseTorch#295] [Polls]
github actions are building a binary from every PR. can you actually see those?

(I'm not very experienced with managing github projects, so I'm not certain you can.)

### Step 1
![noisetorch-pr-actions-01](https://user-images.githubusercontent.com/1178490/170317928-9fd899ed-722a-49ae-a40c-8b2ee9c3ae6c.png)

### Step 2
![noisetorch-pr-actions-02](https://user-images.githubusercontent.com/1178490/170317933-0c49ab03-ff7f-40a3-ad58-e0718a2ca3cf.png)

### Step 3
![noisetorch-pr-actions-03](https://user-images.githubusercontent.com/1178490/170317934-57b02e6c-78da-4f46-8f8b-4dad1021f056.png)

**Comment by CheaterTim** [comment]
for me its `Actions -> Build Dev Artifact -> also build artifact on pushes to master branch (the latest commit?) -> linux_x64`

**Reply by ZyanKLee** [reply]
You are right, the are two ways to get there. When reviewing PRs it send easier for me to go the other way, but both are valid.
Glad you can download it, that was the main question I had.

**Comment by Technetium1** [comment]
Everything seems to be working as intended. Artifacts and the full logs are public.

**Comment by ZyanKLee** [comment]
thanks a lot
